tests/playTests.sh

```diff
@@ -1029,6 +1029,9 @@ then
 fi
 rm -f tmp* dictionary
 
+println "- Test --memory for dictionary compression"
+datagen -g12M -P90 > tmpCorpusHighCompress
+zstd --train -B2K tmpCorpusHighCompress -o tmpDictHighCompress --memory=10K && die "Dictionary training should fail : --memory too low (10K)"
```

Comment:
It seems you are trying to generate a side effect (training failure) to prove that `--memory` is active.

I was wondering if it would be possible to consider a test that checks a successful action of `--memory`, for example by `grep`ing `stderr` and looking for the warning message that `--memory` will trigger ?

Reply:
Good idea, I will add a second test.